commands.go

```diff
@@ -220,6 +220,7 @@ type Cmdable interface {
 	BgRewriteAOF() *StatusCmd
 	BgSave() *StatusCmd
 	ClientKill(ipPort string) *StatusCmd
+	ClientKillByOption(keys ...string) *IntCmd
```

Comment:
I'd rather remove existing `ClientKill` and rename your version `ClientKillByOption` -> `ClientKill`. Could you do that?

Reply:
No, it is not - I did not notice it. Let's keep this as is, but rename `ClientKillByOption` to `ClientKillByFilter` since that is what Redis calls it `CLIENT KILL <filter> <value> ... ... <filter> <value>`. Thanks.